common/kernel/basectx.h

```diff
@@ -213,7 +213,9 @@ struct BaseCtx
 
     NetInfo *getNetByAlias(IdString alias) const
     {
-        return nets.count(alias) ? nets.at(alias).get() : nets.at(net_aliases.at(alias)).get();
+        if(nets.count(alias) > 0) return nets.at(alias).get();
```

Comment:
can you run a `make clangformat` on this?

Reply:
Now done.